broker-daemon/orderbook/index.js

```diff
@@ -85,6 +86,7 @@ class Orderbook {
      */
     const onWatcherSync = () => {
       this.synced = true
+      retries = 0
```

Comment:
do you have a test for this behavior? Sorry if I missed it.

Reply:
Ah good catch! I've added a test for this behavior, however it requires emitting multiple `end` events along with a `sync` event and checking the argument passed to `watchMarket` to get the retry count. If you see a more direct way of testing this behavior I'm happy to update.